scripts/kubectl-workspace-ensure

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 The KCP Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Purpose: ensure that a workload management workspace exists and has
+# the needed APIBindings.
+
```

Comment:
FYI, it would be good to document the usage/semantics.  I see in the implementation that the given pathname can either start with "root" or not, and in the former case it is an absolute path and in the latter case it is a relative path.  I think it would be good to write something like that.